### PR TITLE
show matched options in result

### PR DESCRIPTION
**Pull request by v-i-s-h-n-u-ps**

### CODE REVIEW

File: src/styles.css
Comment (esezen) [inline src/styles.css:1094]: This creates a weird space in certain cases. What do you think about making this `flex` and left aligning it?
image

File: src/components/ResultCard/ResultCard.tsx
Comment (esezen) [inline src/components/ResultCard/ResultCard.tsx:102]: Can we hide this by default and display it only if the consumer activates it via the config? Something like `resultCardOptions.displayMatchedOptions` (feel free to pick a different name)

File: src/components/CioQuiz/quizApiReducer.ts
Comment (esezen) [inline src/components/CioQuiz/quizApiReducer.ts:81]: Lint doesn't like this at the moment. I published a new version for `client-js` with the updated types. You can update the version to get rid of the lint issue

**Review by esezen — CHANGES_REQUESTED** [review]
Thanks for working on this! I left some small comments

File: src/hooks/useQuiz.ts
Comment (esezen) [inline src/hooks/useQuiz.ts:55]: Can you add documentation around this in `src/stories/Quiz/Hooks/Docs/markdown/StateDocs.md`?

File: src/hooks/useQuiz.ts
Comment (esezen) [inline src/hooks/useQuiz.ts:52]: This is not related to this line but can you also update `src/stories/Quiz/Hooks/index.tsx` and `src/stories/Quiz/Hooks/Docs/markdown/CodeExample.md` to match the new format?

File: src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx
Comment (esezen) [inline src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx:14]: Let me know if this doesn't happen for you but the results config is not being fetched for me at the moment

image

File: src/styles.css
Comment (esezen) [inline src/styles.css:976]: I think we can use some bottom margin here.
image

**Review by esezen — CHANGES_REQUESTED** [review]

File: src/hooks/useQuiz.ts
Comment (v-i-s-h-n-u-ps) [reply src/hooks/useQuiz.ts:55]: [9e749d6](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/148/commits/9e749d6bba3f4e510b6fd98bbe0e260357f63cf4)

File: src/hooks/useQuiz.ts
Comment (v-i-s-h-n-u-ps) [reply src/hooks/useQuiz.ts:52]: [9e749d6](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/148/commits/9e749d6bba3f4e510b6fd98bbe0e260357f63cf4)

File: src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx
Comment (v-i-s-h-n-u-ps) [reply src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx:14]: I just relaunched the app and the config is being fetched for me. Can you share the flow where you got the error?

Screenshot 2023-12-26 at 9 52 33 PM

File: src/styles.css
Comment (v-i-s-h-n-u-ps) [reply src/styles.css:976]: [8e8523e](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/148/commits/8e8523eabe4324902949ad899b57d8cbc62d04d7)

File: src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx
Comment (v-i-s-h-n-u-ps) [reply src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx:14]: Also on refresh
Screenshot 2023-12-26 at 10 51 27 PM

**Comment by esezen** [comment]
This is looking good. Let's get a review from @mocca102 and get this merged

File: src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx
Comment (mocca102) [inline src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx:15]: Can we merge this if else block statement with the one below on line 26 to 34 since they use mostly the same conditions?

**Review by mocca102 — APPROVED** [review]
LGTM! left one comment

**Review by esezen — APPROVED** [review]

**Pull request merged**
